src/ChunkedIO.cc

```diff
@@ -377,7 +376,7 @@ ChunkedIO::Chunk* ChunkedIOFd::ConcatChunks(Chunk* c1, Chunk* c2)
 
 void ChunkedIO::Log(const char* str)
 	{
-	RemoteSerializer::Log(RemoteSerializer::LogError, str);
+	//RemoteSerializer::Log(RemoteSerializer::LogError, str);
```

Comment:
Note - this was a kind of lazy compile-fix. ChunkedIO is going to be completely removed with the old value serialization - but still required at the moment.

I was basically too lazy to put  more work in refactoring remote logging out of ChunkedIO given that this cannot be used anymore and that the whole file is going to be removed shortly.